src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --font-family: 'Roboto', Arial, sans-serif;
+}
+
 body {
+  font-family: var(--font-family);
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  margin: 0;
+
+  --header-height: 60px;
+  --blue-color: #00acdc;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header__img {
+  height: 100%;
+  width: 100%;
+  display: inline-block;
+}
+
+.menu {
+  align-items: center;
+  display: flex;
+}
+
+.menu__list {
+  padding: 0;
   margin: 0;
+  display: flex;
+}
+
+.menu__link {
+  text-decoration: none;
+  text-transform: uppercase;
+  white-space: nowrap;
+  text-align: center;
+  display: block;
+  color: inherit;
+  line-height: var(--header-height);
+}
+
+.menu__link.is-active,
+.menu__link:hover {
+  color: var(--blue-color);
+}
+
+.menu__link.is-active {
+  position: relative;
+}
+
+.menu__link.is-active::after {
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  left: 0;
+  background-color: var(--blue-color);
+  bottom: 0;
+  position: absolute;
+  content: '';
+}
+
+.menu__item {
+  margin-right: 20px;
+}
+
+.menu__item:last-child {
+  margin-right: 0;
 }
```

Comment:
Be consistent with your vertical margins. You've added a right margin to .menu__item which is horizontal, but the principle of consistency applies to all margins. Consider if there's a more consistent way to handle spacing between menu items, such as using the `gap` property on the parent flex container.